.cargo/audit.toml

```diff
@@ -0,0 +1,2 @@
+[advisories]
```

Comment:
Can you do as in the dev branch and put this in the workflows folder ?

Reply:
Sure :+1: